pkg/helm/charts.go

```diff
@@ -199,3 +206,14 @@ func parseReq(s string) (*Requirement, error) {
 		Version: *ver,
 	}, nil
 }
+
+// parseReqName parses a name from a string of the format `repo/name` or `name`
+func parseReqName(s string) string {
```

Comment:
It seems like `parseReqName` is only ever called with an argument of the form `name@repo` as `r.Chart` looks like this.

Any reason this function also handles plain `name`?

Reply:
I'm just being defensive in the case a chart name didn't have a `repo/` prefix, I'm not sure how the helm community does it across the board.

Reply:
@sh0rez Does that make sense? I'm happy to simplify it if that case won't happen. Just let me know.

Reply:
The use of `repo/name` promotes making a clear distinction which Chart one refers to, I would be in favor of enforcing this.

Reply:
Gotcha. I removed the check.

Reply:
Thanks @justinwalz!